src/main/java/uk/gov/hmcts/cmc/claimstore/utils/DocumentNameUtils.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.cmc.claimstore.utils;
+
+import static java.lang.String.format;
+
+public class DocumentNameUtils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1118)